notebooks/238-dolly-2-instruction-following/238-dolly-2-instruction-following.ipynb

```diff
@@ -0,0 +1,483 @@
+{
```

Comment:
<p>As we discussed, I think it could be reasonable to use OpenVINO API to get list of supported devices and populate UI list with it. On Mac model is not working now, but even if it is - there will be no GPU anyways...</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1041/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='12'/>